### ISSUE TITLE
pm25_Levels is not defined

### ISSUE THREAD

**Issue by rchen14b** [smallwood69/homebridge-cgllc-airmonitor-s1#6]
I got this error when running this plugin on my homebridge server. Any help would be great! Already got the token from MI home APP.

[9/27/2021, 9:16:17 AM] ReferenceError: pm25_Levels is not defined
    at new ClearGrassAirMonitor (/homebridge/node_modules/homebridge-qingping-airmonitor-cgs1/index.js:40:10)
    at /usr/local/lib/node_modules/homebridge/src/server.ts:350:50
    at Array.forEach (<anonymous>)
    at Server.loadAccessories (/usr/local/lib/node_modules/homebridge/src/server.ts:270:29)
    at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:162:12)v

**Comment by smallwood69** [comment]
please check your home bridge  version.
currently I'm using v1.3.5

**Comment by almone** [comment]
I have the same issue.
Already got token.
Homebridge version: 1.3.5

**Comment by rchen14b** [comment]
@smallwood69  Hi, thanks for your response. I'm using 1.3.8 now and still having the same issue.
Here is the log after I upgraded to 1.3.8:
:37:50 AM] [Config] Initializing config platform...
[11/24/2021, 7:37:50 AM] [Config] Running in Service Mode
[11/24/2021, 7:37:50 AM] [Google Smart Home] Initializing google-smarthome platform...
[11/24/2021, 7:37:50 AM] [Nest] Initializing Nest platform...
[11/24/2021, 7:37:50 AM] Loading 2 accessories...
[11/24/2021, 7:37:50 AM] [Ryan's PC] Initializing NetworkDevice accessory...
[11/24/2021, 7:37:50 AM] [Office Air Monitor] Initializing ClearGrassAirMonitor accessory...
[11/24/2021, 7:37:51 AM] ReferenceError: pm25_Levels is not defined
    at new ClearGrassAirMonitor (/homebridge/node_modules/homebridge-qingping-airmonitor-cgs1/index.js:40:6)
    at /usr/local/lib/node_modules/homebridge/src/server.ts:350:50
    at Array.forEach (<anonymous>)
    at Server.loadAccessories (/usr/local/lib/node_modules/homebridge/src/server.ts:270:29)
    at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:162:12)
[11/24/2021, 7:37:51 AM] Got SIGTERM, shutting down Homebridge...

**Comment by smallwood69** [comment]
please show me your ClearGrassAirMonitor config.

I don't understand. I'm not use 'pm25_Levels' variable

**Comment by ckleinastro** [comment]
I am having this problem as well. 
This is the relevant portion of my hombridge config.json, with the token blanked out:
        {
            "accessory": "ClearGrassAirMonitor",
            "name": "Air Monitor",
            "ip": "192.168.86.94",
            "token": "-------",
            "showTemperature": true,
            "showHumidity": true,
            "showAirQuality": true,
            "showCo2": true,
            "co2_Threshold": 1000
        }

**Comment by ckleinastro** [comment]
I uninstalled, and then re-installed the plugin and now the homebridge boots fully and the Air Monitor device appears for me in homebridge. I think the problem is that I originally installed the ClearGrass plugin from @sparanoid. It looks just like this one, and is maybe a different version(?). I "re-installed" this one from @smallwood69 and that's what led to it now working. The accessory config json in my previous comment is what I used to set it up successfully.

Thanks for any further investigation, and thanks for this plug-in.
